@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -645,7 +645,9 @@ export class MapControls extends THREE.EventDispatcher {
     }
 
     private easeOutCubic(startValue: number, endValue: number, time: number): number {
-        return startValue + (endValue - startValue) * (--time * time * time + 1);
+        return Math.abs(endValue - startValue) < EPSILON
```

Comment:
I think the check should instead occur in the zoom implementation, and on the absolute values and not the relative ones used here. The reason is that `easeOutCubic` is used for both zooming and panning, and that `endValue` and `startValue` are always between 0 and 1.

Reply:
I guess it's a matter of taste, but since this problem can also appear with every action that uses this function (if you limit the positions you can pan in the X axis, for example), I believe that having a ``snap-to-value`` behaviour is quite acceptable.

Reply:
I as well think this logic should not be in easeOutCubic, unless you rename that function.
Btw., can we move this function to some utils b/c I saw that Sergii was re-implementing this function already for the building animation.

Reply:
Ok, I'll keep the change outside of the function. And maybe we can move it outside to Utils, but a bit outside of the scope of this bugfix ticket.

Reply:
okay